tickets/src/main/java/no/unit/nva/publication/ticket/TicketDto.java

```diff
@@ -77,13 +78,13 @@ public abstract class TicketDto implements JsonSerializable {
     private final Username finalizedBy;
     @JsonProperty(FINALIZED_DATE_FIELD)
     private final Instant finalizedDate;
+    @JsonProperty(AVAILABLE_INSTITUTIONS_FIELD)
+    private final Set<URI> availableInstitutions;
 
-    @SuppressWarnings({"PMD.ExcessiveParameterList"})
-    protected TicketDto(URI id, SortableIdentifier identifier, TicketDtoStatus status, List<MessageDto> messages,
+    protected TicketDto(SortableIdentifier identifier, TicketDtoStatus status, List<MessageDto> messages,
                         Set<User> viewedBy, Username assignee, SortableIdentifier publicationIdentifier, User owner,
                         URI ownerAffiliation, Username finalizedBy, Instant finalizedDate, Instant createdDate,
-                        Instant modifiedDate) {
-        this.id = id;
+                        Instant modifiedDate, Collection<URI> availableInstitutions) {
```

Comment:
Removing id as input parameter to constructor, we can create it based on identifier and resourceIdentifier provided in constructor.